platform/linuxbsd/x11/display_server_x11.h

```diff
@@ -380,6 +380,7 @@ class DisplayServerX11 : public DisplayServer {
 	void _window_changed(XEvent *event);
 
 public:
+	bool close_top_popup();
```

Comment:
This function doesn't need to be public, so it would be better to make it private.
Private functions in this C++ code base are supposed to start with an underscore "`_`".

Since the return value is not used, the function  definition should be `void` instead of `bool`.

Reply:
The current solution is almost certainly not the right one so don't think this detail is something to fix at the moment